models/promise.js

```diff
@@ -30,8 +30,9 @@ sequelize.authenticate()
       //whom: { type: Sequelize.STRING  }, // to whom are you promising
       tini: { type: Sequelize.INTEGER }, // unixtime that promise was made
       tdue: { type: Sequelize.STRING },
+      // TODO: i don't think we need a field for the domain. we can parse it
```

Comment:
can't think of a good 4 letter abbreviation? 😜 

i think it's useful as a field if we want to query against it later (and it really doesn't 'cost' anything/there's not really a downside to storing a little more data?)